restaurants.json

```diff
@@ -0,0 +1,39 @@
+{
+  "restaurants":[
+    {
+      "id": 0,
+      "name": "Vasilki",
+      "address": "Minsk",
+      "work time": "Monday-Sunday: 08:00 - 23:45",
+      "phone number": "+375297777777"
+    },
+    {
+      "id": 1,
+      "name": "Mama Doma",
+      "address": "Minsk",
+      "work time": "Monday-Sunday: 10:00 - 22:00",
```

Comment:
it is better to avoid spaces in names, e.g. use `workTime`, `work_time`

Reply:
fixed